docs/scos/dev/technical-enhancement-integration-guides/integrating-product-cms-blocks.md

```diff
@@ -24,122 +24,118 @@ related:
+     */
+    protected function getCmsBlockUpdatePlugins(): array
     {
         return array_merge(parent::getCmsBlockUpdatePlugins(), [
-            new CmsBlockProductConnectorUpdatePlugin()
+            new CmsBlockProductAbstractUpdatePlugin(),
         ]);
     }
 }
 ```
 
-4. Register the collector plugin by adding  `CmsBlockProductConnectorCollectorPlugin` to the Collector dependency provider `\Pyz\Zed\Collector\CollectorDependencyProvider`.
+4. Optional: To show which product abstracts are assigned to a block on a block view page, register the product list plugin by adding `CmsBlockProductAbstractListViewPlugin` to the CMS Block GUI dependency provider:
 
-```php
-<?php
-namespace Pyz\Zed\Collector;
-
-...
-
-class CollectorDependencyProvider extends SprykerCollectorDependencyProvider
-{
-    public function provideBusinessLayerDependencies(Container $container)
-    {
-        ...
-        $container[self::STORAGE_PLUGINS] = function (Container $container) {
-            return [
-                ...
-                CmsBlockProductConnectorConstants::RESOURCE_TYPE_CMS_BLOCK_PRODUCT_CONNECTOR => new CmsBlockProductConnectorCollectorPlugin(),
-            ];
-        };
-    }
-}
-```
-
-5. Register the product list plugin (optional).
-To show which product abstracts are assigned to a block on a block view page, add `CmsBlockProductAbstractListViewPlugin` to the CMS Block GUI dependency provider.
+**src/Pyz/Zed/CmsBlockGui/CmsBlockGuiDependencyProvider.php**
 
 ```php
 <?php
+
 namespace Pyz\Zed\CmsBlockGui;
 
-...
+use Spryker\Zed\CmsBlockGui\CmsBlockGuiDependencyProvider as SprykerCmsBlockGuiDependencyProvider;
+use Spryker\Zed\CmsBlockProductConnector\Communication\Plugin\CmsBlockProductAbstractListViewPlugin;
 
-class CmsBlockGuiDependencyProvider extends CmsBlockGuiCmsBlockGuiDependencyProvider
+class CmsBlockGuiDependencyProvider extends SprykerCmsBlockGuiDependencyProvider
 {
-    ...
-
     /**
-    * @return array
-    */
-    protected function getCmsBlockViewPlugins()
+     * @return array<\Spryker\Zed\CmsBlockGui\Communication\Plugin\CmsBlockViewPluginInterface>
+     */
+    protected function getCmsBlockViewPlugins(): array
     {
-       return array_merge(parent::getCmsBlockViewPlugins(), [
-           new CmsBlockProductAbstractListViewPlugin(),
-       ]);
+        return array_merge(parent::getCmsBlockViewPlugins(), [
+            new CmsBlockProductAbstractListViewPlugin(),
+        ]);
     }
 }
+
 ```
 
-6. Register the block list to the product abstract view (optional).
-To show which blocks are assigned to a product abstract on a product abstract view page, add `CmsBlockProductAbstractBlockListViewPlugin` to the Product Management dependency provider.
+5. Optional: To show which blocks are assigned to a product on a product abstract view page, register the block list to the product abstract view by adding `CmsBlockProductAbstractBlockListViewPlugin` to the Product Management dependency provider:
```

Comment:
`register the block list to the product abstract view` seems incorrect. Anything we can improve here?

Reply:
I think we can simply remove this line